src/component/CodeCardBox.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {Text, FlatList, Image, View} from 'react-native';
+
+const PCodeCardBox = props => {
+  const {style, img, codinglang} = props;
```

Comment:
when you create a component that component already has sample style it means dose not need all of them send style as param only when need who want to create something different in that case you accept merge style params  with component params